SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security issues to `info@inveniosoftware.org`
```

Comment:
Not sure why this PR was not yet merged... to facilitate, reflecting comment as a suggestion:

```suggestion
Please report security issues to `info@inveniosoftware.org`.
See our [Security Policy](https://inveniordm.docs.cern.ch/releases/security-policy/) for more information.
```